sdk/ai/ai-projects/src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,51 @@
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ConnectionsOperations } from "../connections/index.js";
+
+/** 
+ * Telemetry options
+ */
+export interface TelemetryOptions {
+    enableContentRecording: boolean;
+}
+
+const telemetryOptions: TelemetryOptions & { connectionString: string | unknown } = {
+    enableContentRecording: false,
+    connectionString: null
```

Comment:
I definitely worry when I see connectionString anywhere let alone in a concept tied to telemetry - what is the usecase here and how easy is it for someone to accidentally send their connection string over the wire without realizing it? A sample would help me understand a little more

Reply:
I have updated the sample, this is in line with the Python and C# libraries for AI Projects